northstar/src/runtime/console.rs

```diff
@@ -357,15 +357,16 @@ impl Console {
                     let (reply_tx, reply_rx) = oneshot::channel();
 
                     // Send the request to the runtime
-                    event_tx
-                        .send(Event::Console(request, reply_tx))
-                        .await
-                        .expect("Internal channel error on main");
-
-                    // Wait for the reply from the runtime
-                    let response: api::model::Response = reply_rx
-                        .await
-                        .expect("Internal channel error on client reply");
+                    trace!("{:?} -> event loop", request);
+                    let response = match event_tx.send(Event::Console(request, reply_tx)).then(|_| reply_rx).await {
+                        Ok(response) => response,
+                        Err(e) => {
+                            // This could happen if the runtime is in process of shutting down
+                            warn!("Failed to get response from event loop: {}", e);
+                            break;
+                        }
+                    };
+                    trace!("{:?} <- event loop", response);
```

Comment:
If the connections are closed, before everything else it shut down the `expects` are fine. This patch logs things but don't fail on the internal bug when the main loop is down or the runtime core doesn't respond.

Reply:
The problem are the connections that are at this point waiting for a response. They won't close and therefore won't permit that the console is finally closed. That's the actual issue here.

Reply:
Is the "waiting for a response" uninterruptible? Should be doable with the `CancelationToken` you pass into the connection loop.